datafusion/datasource/src/file_scan_config.rs

```diff
@@ -192,6 +189,9 @@ pub struct FileScanConfig {
     /// Expression adapter used to adapt filters and projections that are pushed down into the scan
     /// from the logical schema to the physical schema of the file.
     pub expr_adapter_factory: Option<Arc<dyn PhysicalExprAdapterFactory>>,
+    /// Unprojected statistics for the table (file schema + partition columns).
+    /// These are projected on-demand via `projected_stats()`.
+    pub(crate) statistics: Statistics,
```

Comment:
nit is that this field has a different visibility (`pub(crate)`) than all the other fields